tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/Tainted/DefaultTaintedMapTests.cs

```diff
@@ -115,7 +115,7 @@ public void GivenATaintedObjectMap_WhenClear_ObjectIsNotRetrieved()
     {
         DefaultTaintedMap map = new();
         var testObject = new object();
-        var source = new Source(12, "name", "value");
+        var source = new Source(SourceType.RequestBody, "name", "value");
```

Comment:
`RequestBody` is 0 and not 12 (which doesnt exist and would throw in the switches)? is this ok?

Reply:
Yes, before they where meaningless numbers. These are unit tests and the origin value is not checked